Confus/Confus/Game.cpp

```diff
@@ -95,6 +107,16 @@ namespace Confus
         m_DeltaTime = (m_CurrentTicks - m_PreviousTicks) / 1000.0;
 
         m_PlayerNode.update();
+        m_Listener.setPosition(m_PlayerNode.getAbsolutePosition());
+
+        auto rot = m_PlayerNode.CameraNode->getRotation();
+        irr::core::quaternion playerRotation(rot);
+
+
+        //Todo: Fix rotations
+        irr::core::vector3df upVector = playerRotation * irr::core::vector3df( 0, 1, 0 );
```

Comment:
I noticed that the sound listener did not get updated and decided to add this code. However since irrlicht does not use Quaternions for rotation I was unable to get the correct up and forward   vectors. The rotation sound is sometimes off now, it might be better to create a card for the next sprint instead of fixing it now?